src/App.tsx

```diff
@@ -134,27 +191,54 @@ const App = () => {
           anchor="right"
           open={open}
           classes={{
-            paper: classes.drawerPaper,
+              paper: classes.drawerPaper,
           }}
         >
           <div className={classes.drawerHeader}>
-            <IconButton onClick={handleDrawerClose}>
+              <IconButton onClick={handleDrawerClose}>
               <ChevronRightIcon />
-            </IconButton>
-            <Typography>ARC :)</Typography>
+              </IconButton>
+              <Typography>ARC :)</Typography>
```

Comment:
Same here, it might be a formatter issue, but it looks like the code below is indented correctly so idk